platform/pulpcore/app/tasks/publisher.py

```diff
@@ -3,11 +3,20 @@
 from celery import shared_task
 
 from pulpcore.app import models
+from pulpcore.app import serializers
 from pulpcore.tasking.services import storage
 from pulpcore.tasking.tasks import UserFacingTask
 
 
 @shared_task(base=UserFacingTask)
+def update(publisher_id, data, *args, **kwargs):
+    instance = models.Publisher.objects.get(id=publisher_id)
+    serializer = serializers.PublisherSerializer(instance, data=data, partial=partial)
```

Comment:
partial needs to be `kwargs.pop('partial')` or added as a keyword argument in the function definition